splunk_otel/symbols.py

```diff
@@ -26,6 +26,7 @@
 
 _SPLUNK_DISTRO_VERSION_ATTR = "splunk.distro.version"
 _SERVICE_NAME_ATTR = "service.name"
+_SPLUNK_SERVICE_NAME_ATTR = "sf_service"
```

Comment:
I don't think other distros use this attribute, feel free to correct me. At least in Node this isn't added.

Reply:
They don't, and this tag name is supposed to be added by the collector. However, it is not being added which makes the metrics to be not filtrable by service name in the dashboards in the UI. This is my attempt to fix this.

Reply:
Are we sure it's supposed to be added by the collector? I'm not seeing this when I do a [search](https://github.com/search?q=repo%3Asignalfx%2Fsplunk-otel-collector%20sf_service&type=code) and I thought it was added by the sfx backend 🤔

Reply:
The jury is still out on this (whether it is supposed to be added or not), all dug out so far is that it is not being added for multiple instrumentations, not just for splunk-otel-python right now. We might need a global fix in the future, and in the meantime I want to make sure our customers can still build meaningful dashboards for their python apps.

Reply:
I think that before merging this change, we should get to the bottom of whether (and if so, why) sending sf_service is needed. My guess is that it's not needed as evidenced by other libraries not sending it, but it would be good to confirm the behavior.

Reply:
Removed this tag for now so we could continue the discussion at a later time.